tox.ini

```diff
@@ -1,20 +1,15 @@
 [tox]
-envlist = py35, py36, py37, py38, flake8
-
-[travis]
-python =
-    3.8: py38
-    3.7: py37
-    3.6: py36
-    3.5: py35
+envlist = py35, py36, py37, py38, py39, py310, flake8
```

Comment:
[IMO] should remove py35 and py36 because it is going to be EOL status

https://endoflife.date/python

Reply:
Thank you for your opinion. 
Honestly speaking, I don't understand what this file exists for. It's also a automatically generated file, actually.

Reply:
This is `tox`'s test environment flag. If you add `pyxy` here, the tests run on python x.y version.
This feature is useful for library development (you can check your library correctly run on several python versions), but it is not necessary for this repo because this is CLI tool.
So my recommendation is omitting `py35-38` to reduce CI running time on GitHub Actions (I'll create it in next PR).